sql/src/test/java/io/crate/metadata/SchemasITest.java

```diff
@@ -180,4 +184,88 @@ public void testClusterTable() throws Exception {
             clusterService.state(), routingProvider, null, null, SessionContext.systemSessionContext()
         ).locations().size(), is(1));
     }
+
+    @Test
+    public void testResolveTable() {
```

Comment:
Would it be possible to split this a bit into more granular cases?

Reply:
If test speed is a concern maybe we could create unit tests for this. `SQLExecutor.builder(..).addTable().addView` should make it easy to retrieve a solid `Schemas` instance